### PR TITLE
Make loadtest app server target configurable

### PR DESCRIPTION
**Pull request by Nico-Sanchez**

### CODE REVIEW

**Review by AminArria — APPROVED** [review]

File: apps/arena_load_test/lib/arena_load_test/socket_handler.ex [hunk 1]
```diff
@@ -55,19 +55,22 @@ defmodule ArenaLoadTest.SocketHandler do
...
   # Private
   defp ws_url(player_id) do
-    host = SocketSupervisor.server_host()
     character = get_random_active_character()
     player_name = "Player_#{player_id}"
 
-    case System.get_env("SSL_ENABLED") do
-      "true" ->
-        "wss://#{host}/join/#{player_id}/#{character}/#{player_name}"
+    case System.get_env("SERVER_HOST") do
```
Comment (AminArria) [inline apps/arena_load_test/lib/arena_load_test/socket_handler.ex:61]: Where is this set?

Comment (Nico-Sanchez) [reply apps/arena_load_test/lib/arena_load_test/socket_handler.ex:61]: woops. I forgot to set the input variable in the yaml file. It's done now!

File: apps/arena_load_test/lib/arena_load_test/socket_supervisor.ex [hunk 2]
```diff
@@ -36,7 +36,6 @@ defmodule ArenaLoadTest.SocketSupervisor do
     end
   end
 
-  def server_host() do
-    System.get_env("SERVER_HOST", "localhost:4000")
-  end
+  def get_server_url("Brazil"), do: "brazil-testing.curseofmirra.com"
+  def get_server_url("Europe"), do: "europe-testing.curseofmirra.com"
```
Comment (agustinesco) [inline apps/arena_load_test/lib/arena_load_test/socket_supervisor.ex:40]: i think we should replace this urls with 
`System.get_env(“BRAZIL_HOST”) and System.get_env(“EUROPE_HOST”)` accordingly

**Review by agustinesco — APPROVED** [review]
LGTM!

**Review by lotuuu — APPROVED** [review]

**Pull request merged**